ui/src/submissions/conferences/containers/__tests__/ExistingConferencesAlertContainer.test.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { Provider } from 'react-redux';
+
+import ExistingConferencesAlertContainer from '../ExistingConferencesAlertContainer';
+import { EXISTING_CONFERENCES_NS } from '../../../../reducers/search';
+import { SEARCH_QUERY_UPDATE } from '../../../../actions/actionTypes';
+import { getStore } from '../../../../fixtures/store';
+import { RANGE_AGGREGATION_SELECTION_SEPARATOR } from '../../../../common/constants';
+
+describe('ExistingConferencesAlertContainer', () => {
```

Comment:
you need to test that `numberOfConferences are passed to the component properly. An example for a different container:

```jsx
  it('passes literature query', () => {
    const store = getStoreWithState({
      search: fromJS({
        namespaces: {
          [LITERATURE_NS]: {
            query: { sort: 'mostcited', q: 'query' },
          },
        },
      }),
    });
    const wrapper = mount(
      <Provider store={store}>
        <CitationSummaryBoxContainer />
      </Provider>
    );
    expect(wrapper.find(CitationSummaryBox)).toHaveProp({
      query: { sort: 'mostcited', q: 'query' },
    });
  });
```